kano_profile/app_progress_helpers.py

```diff
@@ -0,0 +1,35 @@
+# app_progress_helpers.py
+#
+# Copyright (C) 2016 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Implement higher level progress helpers
+
+from kano_profile.apps import load_app_state_variable
+
+
+def has_completed_challenges(app_name):
```

Comment:
The name should describe a bit better the functionality. Perhaps `has_completed_a_challenge`?